spec/requests/odk_form_download_spec.rb

```diff
@@ -85,6 +87,33 @@
           end
         end
       end
+
+      context "for forms with audio prompts" do
+        let(:form) { create(:form, :published, mission: mission, question_types: %w[text integer]) }
+
+        before do
+          form.c[0].question.update!(audio_prompt: audio_fixture("powerup.mp3"))
+          form.c[1].question.update!(audio_prompt: audio_fixture("powerup.wav"))
+        end
+
+        it "should render manifest tags correctly" do
+          get("/m/#{mission.compact_name}/forms/#{form.id}/manifest")
+          expect(response).to be_success
+          assert_select("mediaFile", count: 2)
+
+          assert_select "manifest" do |elements|
+            elements.each do |element|
+              assert_select(element, "filename", text: "powerup_#{form.c[0].question.id.first(5)}.mp3")
+              assert_select(element, "filename", text: "powerup_#{form.c[1].question.id.first(5)}.wav")
+
+              assert_select(element, "hash", text: "e7fe3aa406b8b67209b9d89c0cd50aa8")
+              assert_select(element, "hash", text: "ff2fd1e209465c5bffa784b5c57d84c4")
```

Comment:
This is close but not quite. Good job finding the alternative `assert_select` signature (with `element` first). But this setup does not ensure that the right md5 is with the right filename inside the mediaFile tags.

Can you set it up a bit differently so that it does? Hint: you may not need to iterate. Hint 2: There is only one `manifest` element.

Nice job getting the md5s working!